app/views/application/_language.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if I18n.locale == I18n.default_locale %>
+  <%= link_to "It", { :locale=>'it' } %>
```

Comment:
double quotes.
Also, can we use the `{ locale: "it" }` syntax?

Reply:
Done with fixup in sha `16c1673`